openedx/core/djangoapps/profile_images/views.py

```diff
@@ -35,49 +35,75 @@ def _make_upload_dt():
     return datetime.datetime.utcnow().replace(tzinfo=utc)
 
 
-class ProfileImageUploadView(APIView):
+class ProfileImageView(APIView):
     """
-    **Use Case**
+    **Use Cases**
 
-        * Upload an image for the user's profile.
+        Add or remove profile images associated with the user's account.
 
-          The requesting user must be signed in. The signed in user can only
-          upload his or her own profile image.
+        The requesting user must be signed in.  Users can only add profile
+        images to their own account.  Users with staff access can remove
+        profile images for other user accounts.  All other user can
+        only remove their own profile images.
 
-    **Example Request**
+    **Example Requests**
 
-        POST /api/profile_images/v1/{username}/upload
+        POST /api/user/v1/accounts/{username}/image
 
-    **Example Responses**
+        DELETE /api/user/v1/accounts/{username}/image
 
-        When the requesting user tries to upload the image for a different user, the
-        request returns one of the following responses.
+    **Example POST Responses**
 
-        * If the requesting user has staff access, the request returns an HTTP 403
-          "Forbidden" response.
+        * If the upload could not be performed, the request returns an HTTP 400
+          "Bad Request" response with more information.
 
-        * If the requesting user does not have staff access, the request returns
-          an HTTP 404 "Not Found" response.
+        * If the upload is successful, the request returns an HTTP 204 "No
+          Content" response with no additional content.
 
-        * If no user matches the "username" parameter, the request returns an HTTP
-          404 "Not Found" response.
+        When the requesting user tries to upload the image for a different
```

Comment:
A success response is also possible `when the requesting user tries to upload the image for a different user`.